tico/passes/decompose_group_norm.py

```diff
@@ -155,52 +174,20 @@ def call(self, exported_program: ExportedProgram) -> PassResult:
             pack_shape = [layer_size, norm_size]
 
             with gm.graph.inserting_before(node):
-                layer = graph.call_function(
-                    # Sometimes, `x` has a stride for NHWC, which can't be reshaped with `aten.view.default`.
-                    # TODO Find out how to process such case properly.
```

Comment:
(optional) How about keeping this comment?

Reply:
Good point. Actually, I removed it intentionally as we can just process them when we exactly meet the case.